### PR TITLE
Add attrs_sqlalchemy

### PR DESCRIPTION
**Pull request by vamega**

### CODE REVIEW

**Comment by vamega** [comment]
@rouge8 just letting you know that I plan to add`attrs-sqlalchemy` to [conda-forge](http://conda-forge.github.io). This will allow users to easily install this with [conda](http://conda.pydata.org/docs/) (`conda install -c conda-forge voluptuous`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. If you have any interest in being added as a maintainer (to make new releases for instance), let me know and I'll add you. Otherwise I'm happy to maintain this release channel.

**Comment by rouge8** [comment]
@vamega I have no experience with Conda, so you'd be a better maintainer than I am. :)

**Comment by vamega** [comment]
@rouge8. Thanks for this. Saved me from having to write this myself.

File: recipes/attrs-sqlalchemy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - attrs
+    - sqlalchemy
+
+test:
+  imports:
+    - attrs_sqlalchemy
+
+about:
+  home: https://github.com/GoodRx/attrs_sqlalchemy
+  license: MIT
```
Comment (scopatz) [inline recipes/attrs-sqlalchemy/meta.yaml]: Can you add the `license_file` entry please?

Comment (vamega) [inline recipes/attrs-sqlalchemy/meta.yaml]: Done!

**Comment by scopatz** [comment]
Thanks!

File: recipes/attrs-sqlalchemy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set name = "attrs_sqlalchemy" %}
```
Comment (jakirkham) [inline recipes/attrs-sqlalchemy/meta.yaml]: @vamega the directory name is `attrs-sqlalchemy`, but the recipe name is `attrs_sqlalchemy`. They differ by `-`/`_`. Can you please send a PR to use the same character in both cases?

**Comment by jakirkham** [comment]
So we are holding up conversion of other recipes to feedstocks because of the aforementioned name issue. Am recommending we pull this recipe until it can be properly fixed. Please see PR ( https://github.com/conda-forge/staged-recipes/pull/1825 ). Would be just as happy to merge a fix for the recipe if it can be put together soon.

**Comment by jakirkham** [comment]
As we have pulled this temporarily, have added issue ( https://github.com/conda-forge/staged-recipes/issues/1826 ) for resubmitting with a fix for the aforementioned problem.

**Pull request merged**